src/App/components/Settings.tsx

```diff
@@ -162,6 +162,13 @@ export default function Settings(): JSX.Element {
           }}
           onFileSelected={settings.setOutputDirectory}
         />
+        <Checkbox
+          checked={settings.overwriteOutputFiles}
+          onChange={(event): void => {
+            settings.setOverwriteFile(event.currentTarget.checked);
+          }}
+          label="Overwrite existing file(s)"
```

Comment:
nit: I wouldn't bother with the parenthesis for plural. Its fine in English to use plural for the general case.